libs/model/src/info.cpp

```diff
@@ -313,9 +313,13 @@ nlohmann::json Coverage::toJson() const
 std::shared_ptr<LayerInfo> LayerInfo::fromJson(const nlohmann::json& j, std::string const& layerId)
 {
     try {
+        const auto type = j.value("type", LayerType::Features);
+
         std::vector<FeatureTypeInfo> featureTypes;
-        for (auto& item : j.at("featureTypes")) {
-            featureTypes.push_back(FeatureTypeInfo::fromJson(item));
+        if (type == LayerType::Features) {
```

Comment:
Idea: Save all SourceDataRefs in a cont. buffer. Each referee stores an index and size -> use simfil::ArrayArena. Maybe create a `TinyFixedSizeArrayArena`, where chunks are only `3B index + 1B length`.

Reply:
See PR https://github.com/Klebert-Engineering/simfil/pull/90.